src/main/java/beam/analysis/plots/DeadHeadingAnalysis.java

```diff
@@ -74,8 +74,8 @@ private static String getLegendText(String graphName, int i, int bucketSize) {
 
     @Override
     public void processStats(Event event) {
-        if (event instanceof PathTraversalEvent || event.getEventType().equalsIgnoreCase(PathTraversalEvent.EVENT_TYPE))
-            processDeadHeading(event);
+        if (event instanceof PathTraversalEvent)
```

Comment:
Why the `||` removal? I don't know what the reason for this would have been in the first place, but wanted to make sure this was done with the appropriate due diligence of logic removal

Reply:
I won't repeat the comment for further changes just like this - but the same would go for all.

Reply:
I removed it because it looks kind of weird. `PathTraversalEvent.EVENT_TYPE` is from `PathTraversalEvent`, that means that `event` has to be `PathTraversalEvent` reference (otherwise there is a bug here)

Reply:
Thinking about this - if everything is supposed to be PTE then should we add this to the type signature of the class itself? NBD otherwise.